app/views/pages/moving-guide/claims.html.erb

```diff
@@ -0,0 +1,198 @@
+          </tr>
+          <tr>
+            <th scope="row">Between 9 Months and 2 years after Delivery Day</th>
+            <td>Contact the <%= link_to 'Military Claims Office', page_path( 'customer-service', anchor: 'claims') %> for help receiving partial replacement value for any lost or damaged items not previously submitted.</td>
+          </tr>
+          <tr>
+            <th scope="row">60 Days after Claim Submission</th>
+            <td>Your moving company (Transportation Service Provider) should pay, deny, or make an alternate to your claim within 60 days after your claim is final.</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <table>
+        <thead>
+          <tr>
+            <th colspan="2">Important Dates WITHOUT Loss/Damage Report</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">Up to 75 Days after Delivery Day</th>
+            <td>Sign-In to the <%= link_to 'Defense Personal Property System (DPS)', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> and submit itemized claims for any lost or damaged items for full replacement value.</td>
+          </tr>
+          <tr>
+            <th scope="row">Between 75 Days and 2 years after Delivery Day</th>
+            <td>Contact the <%= link_to 'Military Claims Office', page_path( 'customer-service', anchor: 'claims') %> for help receiving partial replacement value for any lost or damaged items not previously submitted.</td>
+          </tr>
+          <tr>
+            <th scope="row">60 Days after Claim Submission</th>
+            <td>Your moving company (Transportation Service Provider) should pay, deny, or make an alternate to your claim within 60 days after your claim is final.</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <div class="usa-media_block pro-tip">
+        <%= image_tag 'icons/pro-tip.svg', alt: 'lightbulb icon', class: 'usa-media_block-img' %>
+        <div class="usa-media_block-body">
+          <strong>Pro-Tip:</strong> If you file a Loss/Damage Report, you can submit claims for any additional lost or damaged items discovered after the initial 75 Day window… even if it wasn’t included on the original Lost/Damage Report.
+        </div>
+      </div>
+    </div>
+
+    <div class="what-to-expect-section" id="claim">
+      <h4>Itemized Claim</h4>
+
+      <p>Regardless of whether you submit a Loss & Damage Report, you must submit an itemized claim in the <%= link_to 'Defense Personal Property System (DPS)', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> for every item that was lost or damaged during the moving process.</p>
+
+      <p>Before logging into the system, save yourself some time by acquiring as many of artifacts below… the more details you can provide the better. If you have before and after photos or receipts of the item that will <strong>significantly help</strong> throughout the process.</p>
```

Comment:
"Before logging into the system" As in, before logging in to submit a claim? Or should this actually be, "Before pack day"?

"save yourself some time by acquiring as many of artifacts below…"

Should probably add the implied "as you can" to the end here. Also, the ellipses are unnecessary.

"If you have before and after photos or receipts of the item that will significantly help throughout the process."

Add a comma between "item" and "that".